### ISSUE TITLE
doc bug:  cudnn install paths

### ISSUE THREAD

**Issue by gciccarelli3** [Theano/Theano#5156]
http://deeplearning.net/software/theano_versions/dev/library/sandbox/cuda/dnn.html

date accessed:  Oct 30, 2016, 11:28am eastern time.

I believe the 3rd line of the example code for paths should have LIBRARY_PATH not LD_LIBRARY_PATH at the end.  

#### current doc  
    
    export LD_LIBRARY_PATH=/home/user/path_to_CUDNN_folder/lib64:$LD_LIBRARY_PATH
    export CPATH=/home/user/path_to_CUDNN_folder/include:$CPATH
    export LIBRARY_PATH=/home/user/path_to_CUDNN_folder/lib64:$LD_LIBRARY_PATH

#### proposed change  
    
    export LIBRARY_PATH=/home/user/path_to_CUDNN_folder/lib64:$LIBRARY_PATH

**Comment by nouiz** [comment]
Exact. You can make a pr is you want to be in the contributor list.
Otherwise I'll make the fix next week.

Thanks for the report.
